DER/DerivativesContracts/Options.rdf

```diff
@@ -620,45 +512,46 @@
 	<owl:Class rdf:about="&fibo-fbc-fi-fi;Option">
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasCalculatedMarketValue"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasOptionHolder"/>
+				<owl:onClass rdf:resource="&fibo-der-drc-opt;OptionHolder"/>
 				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
-				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;OptionPremium"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-der-der-opt;hasExercisePrice"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+				<owl:onProperty rdf:resource="&fibo-der-drc-opt;hasCalculatedMarketValue"/>
```

Comment:
why the calculated value? That's not what anyone will never pay as a premium - that's determined by supply and demand in the order book.

Reply:
@rivettp This came from Nordea and others - they keep a running calculated market value on their books for managing the valuation of a portfolio at the end of the day, in particular for options, used for managing risk.  There are likely other cases where this kind of thing is needed, but it's especially important for daily valuation requirements (regulatory) for options.

Reply:
Should then link to the calculation method (Black Scholes etc)?